modules/layers/src/scatterplot-layer/scatterplot-layer.ts

```diff
@@ -273,8 +273,15 @@ export default class ScatterplotLayer<DataT = any, ExtraPropsT extends {} = {}>
     // a square that minimally cover the unit circle
     const positions = [-1, -1, 0, 1, -1, 0, 1, 1, 0, -1, 1, 0];
 
+    const shaders = this.getShaders();
+    const renderer = this.context.device.info.renderer;
+    if (renderer.includes('Xclipse 920')) {
+      shaders.defines = {...shaders.defines};
+      shaders.defines.XCLIPSE920 = 1;
```

Comment:
Better move to https://github.com/visgl/luma.gl/blob/8.5-release/modules/shadertools/src/lib/platform-defines.js ?

Reply:
Thought of it, though hoped that keeping the logic localized might be a reasonable option.
All the other shaders and many WebGL samples work fine with this graphics so hopefully the w/a will not proliferate.
It's a specific device version, so I'm not sure it should be generalized to a platform level. Cannot tell at the moment if future Xclipse SKUs (if any) would be also affected.
Also taking it next step further I'm not sure luma aspires to contain a catalogue of graphics units.